### ISSUE TITLE
508-defect-2 [SCREENREADER] Hours column in "Call Us" table not announcing correctly

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#20239]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah Gelman

## Details

`Monday through Friday, 8:00 a.m. to 7:00 p.m. ET`

The `a.m./p.m.` in the above text does not announce correctly. It needs the periods removed to be `am/pm`
The `ET` in the above text does not announce correctly. It needs an aria-label to be announced as `Eastern Time'

## Acceptance Criteria

- [ ] Remove periods from `a.m./p.m.`
- [ ] Wrap ET (or other time zone) in aria-label to correctly announce it's full time zone

## Proposed Solution (if known)

`a.m./p.m.` should be `am/pm`
'ET' should be '<span aria-label="Eastern Time">ET</span>`

**Comment by RLHecht** [comment]
@noahgelman Our VA.gov style guide has us using periods in a.m. and p.m. across the site, so we wouldn't remove it on this page. Is there something that should/could be done within the source code to fix this? if so then it would go to the public websites team.

**Comment by noahgelman** [comment]
@RLHecht If the style guide is using the periods then they can be wrapped in a span with an aria label to announce it correctly.

`<span aria-label="am">a.m.</span>`

**Comment by RLHecht** [comment]
I don't know if this is something that can be done on the FE globally for each time we have a.m./p.m. but I think it's something PW would need to look into.

**Comment by brianalloyd** [comment]
@kelsonic looking into sitewide impact.  Might have some questions on implementation and taxonomy.

**Comment by brianalloyd** [comment]
PW

**Comment by joshkimux** [comment]
@laflannery

**Comment by laflannery** [comment]
Screen readers users are user to things sometimes being read a little bit funny. Also, how a screen reader ammounces something often depends on the reader/browser combination as well as their settings. We don't want to try and force or override everything by overusing aria-labels. This is an instance where folks should be able to understand the time and what is being said and specific labels shouldn't be necessary.
